src/views/Detail.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-container>
+    <v-row dense>
+      <v-col cols="12">
+        <AppDetail v-bind:detailed="detailed" />
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import AppDetail from '@/components/app-detail/app-detail.vue';
+import api from '@/api';
+
+export default {
+  name: 'Detail',
+  components: {
+    AppDetail,
+  },
+  data() {
+    return {
+      detailed: {},
+    };
+  },
+  async created() {
+    this.detailed = await api.lookup(Number(this.$route.params.id));
```

Comment:
Please use parseInt for this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseInt

Reply:
I added an early validation on what's coming from the router params